sml_builder/utils.py

```diff
@@ -14,3 +15,13 @@ def convert_term(value):
 def _page_not_found(error):
     print(error)
     abort(404)
+
+
+def get_feature_config(feature_name: str):
```

Comment:
We should have a one-liner comment to explain the feature config and why we get it.